modules/lndr_example/lndr.example.inc

```diff
@@ -26,66 +33,61 @@ function lndr_example($form, &$form_state) {
 }
 
 /**
- * Validation handler
+ * Validation handler.
+ *
  * @param $form
  * @param $form_state
  */
 function lndr_example_validate($form, &$form_state) {
-  if (!valid_url($form_state['values']['lndr_example_url'], true)) {
+drush_print_r(__FUNCTION__);
+
+  if (!valid_url($form_state['values']['lndr_example_url'], TRUE)) {
     form_set_error('lndr_example_url', t('Please enter a valid URL'));
   }
   $internal_url = drupal_parse_url($form_state['values']['lndr_example_url']);
   if (substr($internal_url['path'], 0, strlen(LNDR_BASE_PUBLISHED)) != LNDR_BASE_PUBLISHED) {
-    form_set_error('lndr_example_url', t('only lndr specific URLs can be used, it should start with @url', array('@url'=> LNDR_BASE_PUBLISHED)));
+    form_set_error('lndr_example_url', t('only lndr specific URLs can be used, it should start with @url', array('@url' => LNDR_BASE_PUBLISHED)));
   }
 }
 
 /**
- * Submission handler
+ * Submission handler.
+ *
  * @param $form
  * @param $form_state
  */
 function lndr_example_submit($form, &$form_state) {
+drush_print_r(__FUNCTION__);
+
   drupal_goto('examples/lndr/render', array('query' => array('url' => $form_state['values']['lndr_example_url'])));
 }
 
 /**
- * Simple page to show how lndr landing page looks in Drupal
+ * Simple page to show how lndr landing page looks in Drupal.
+ *
  * @return string
  */
 function lndr_example_render() {
+drush_print_r(__FUNCTION__);
+
   $query = drupal_get_query_parameters();
   if (!isset($query['url'])) {
     return 'Invalid url passed';
   }
-  // Add trailing slash
+  // Add trailing slash.
   if (substr($query['url'], -1, 1) !== '/') {
     $query['url'] .= '/';
   }
   print _lndr_import_page($query['url']);
 }
 
 /**
- * emulate the lndr web service call
+ * Emulate the lndr web service call.
  */
 function lndr_example_service() {
+drush_print_r(__FUNCTION__);
+
   $path = drupal_get_path('module', 'lndr') . '/lndr_test.json';
```

Comment:
Since you have moved the local data test into the debug module, should we move the lndr_test.json into the lndr_debug module directory?